web/src/components/MultiFormPage/MultiFormPage.jsx

```diff
@@ -21,29 +22,36 @@ const steps = [
 ];
 const MultiFormPage = () => {
   const [currentStepIndex, setCurrentStepIndex] = useState(0);
-  const [formData, setFormData] = useState({
-    personalInfo: {},
-    professionalSummary: {},
-    transferableExperience: {},
-    education: [
-      {
-        institution: '',
-        program: '',
-        startDate: '',
-        endDate: '',
-      },
-    ],
-    projects: [
-      {
-        name: '',
-        description: '',
-        deployedWebsite: '',
-        githubLink: '',
-      },
-    ],
-    profileVsJobCriteria: {},
+  const [formData, setFormData] = useState(() => {
+    const savedData = getFormData();
+    return {
+      personalInfo: savedData.personalInfo || {},
+      professionalSummary: savedData.professionalSummary || {},
+      transferableExperience: savedData.transferableExperience || {},
+      education: savedData.education || [
+        {
+          institution: '',
+          program: '',
+          startDate: '',
+          endDate: '',
+        },
+      ],
+      projects: savedData.projects || [
+        {
+          name: '',
+          description: '',
+          deployedWebsite: '',
+          githubLink: '',
+        },
+      ],
+      profileVsJobCriteria: savedData.profileVsJobCriteria || {},
+    };
```

Comment:
When loading this data I imagine there is a brief loading flash, we can maybe consider adding a loading state here to handle this flash better